hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/authority/DefaultApprover.java

```diff
@@ -88,7 +88,8 @@ public  X509CertificateHolder sign(
       Date validTill,
       PKCS10CertificationRequest certificationRequest,
       String scmId,
-      String clusterId) throws IOException, OperatorCreationException {
+      String clusterId) throws IOException,
+      OperatorCreationException {
```

Comment:
Nit: can we omit this formatting-only change?

Reply:
Done